model_clone/tests/test_clone_mixin.py

```diff
@@ -717,6 +717,7 @@ def test_cloning_many_to_one(
 
         book.page_set.set([page])
         edition_clone = edition.make_clone()
+        edition_clone.refresh_from_db()
```

Comment:
Can we make a similar change to all test methods prior to checking the expected state of the new object?